about.html

```diff
@@ -13,52 +13,74 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+      <header class='header-content'>
+        <nav>
+          <img src="img/lambda-black.png" alt="Lambda Logo">
+          <a href="./index.html">Home</a>
+          <a href="#">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+        <img src="img/jumbo-about.png" alt="Jumbo-about image">
+      </header>
+        
+      <section class='about-content'> 
+        <h2>About</h2>
 
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </section>
 
-    <img src="img/about-plan.png" alt="strategy">
+      <main>
+        <section class='flex-card'>
+          <img src="img/about-plan.png" alt="strategy">
     
-    Strategy
+          <h2>Strategy</h2>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
+          <button>Learn More</button>
+        </section>
 
```

Comment:
You want to remove the white space, from your code. Though it won't make a difference for your web page it does add more unnecessary lines to your code.